docs/source/daml2js/index.rst

```diff
@@ -1,44 +1,42 @@
 .. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-``daml2js`` - TypeScript code generator for DAML
-################################################
+DAML to JavaScript code generation
+##################################
 
-daml2js : TypeScript of DAML
-============================
+Code generation of DAML JavaScript bindings (with TypeScript typings)
+=====================================================================
 
 Introduction
 ------------
 
-``daml2js`` generates TypeScript for interacting with DAML via the `json-api <../json-api/index.html>`_ .
+The command ``daml codegen js`` generates JavaScript for interacting with DAML via the `json-api <../json-api/index.html>`_ .
 
-``daml2js`` inputs are compiled DAML modules. ``daml2js`` outputs are TypeScript source files containing definitions modeling the DAML entities found.
+Inputs to the command are compiled DAML modules. Outputs are JavaScript packages and TypeScript typings for definitions modeling the DAML entities found in the DAML modules.
 
-The code ``daml2js`` generates uses the library `@daml/types <https://github.com/digital-asset/daml/tree/master/language-support/ts/daml-types>`_.
+The generated packages use the library `@daml/types <https://github.com/digital-asset/daml/tree/master/language-support/ts/daml-types>`_.
 
 Usage
 -----
 
-``daml2js`` is invoked via the DAML SDK assistant.
+In outline, the command to generate JavaScript and TypeScript typings from DAML is ``daml codegen js DAR -o OUTDIR`` where ``DAR`` is the path to a DAR file (generated via ``daml build``) and ``OUTDIR`` is a directory where you want the artifacts to be written.
 
-In outline, the command to generate JavaScript and TypeScript typings from DAML is ``daml codegen ts DAR -o OUTDIR`` where ``DAR`` is the path to a DAR file (generated via ``daml build``) and ``OUTDIR`` is a directory where you want the JavaScript to be written.
-
-Here's a complete example that generates TypeScript from a project produced from the standard "skeleton" template.
+Here's a complete example on a project built from the standard "skeleton" template.
 
 .. code-block:: bash
    :linenos:
 
    daml new my-proj skeleton # Create a new project based off the skeleton template
    cd my-proj # Enter the newly created project directory
    daml build  # Compile the project's DAML files into a DAR
-   daml codegen ts .daml/dist/my-proj-0.0.1.dar -o daml2js # Generate script bindings from the DAR
+   daml codegen js -o daml2js .daml/dist/my-proj-0.0.1.dar # Generate JavaScript packages in the daml2js directory
```

Comment:
```suggestion
   daml codegen js -o dam.js .daml/dist/my-proj-0.0.1.dar # Generate JavaScript packages in the daml.js directory
```
and adapt wherever we refence this directory.

Reply:
ok